app/Http/Controllers/Bible/LibraryController.php

```diff
@@ -504,6 +504,12 @@ public function volume()
                     return $item->english_name;
                 });
 
+            foreach ($filesets as &$fileset) {
+                if ($fileset->secondary_file_type === 'zip') {
```

Comment:
@bradflood it will validate if the fileset is .zip file before to set the audio_zip_path property.